python/libraries/bimvee/importNumpy.py

```diff
@@ -59,7 +59,28 @@ def importNumpy(filePathOrName, **kwargs):
     outDict['info']['filePathOrName'] = filePathOrName
 
     # Importing events
-    events = np.load(filePathOrName)
+    flow = np.load(filePathOrName)
+    outDict['data']['opticFlow'] = {}
+    outDict['data']['opticFlow']['flowMap'] = {}
+    outDict['data']['opticFlow']['flowMap']['flowMaps'] = np.swapaxes(flow, 1, 3)
+    step = 0.03
+    ts = np.arange(0, step * len(flow), step)
+    outDict['data']['opticFlow']['flowMap']['ts'] = ts
+
+    gt_file = os.path.join(os.path.dirname(filePathOrName), 'outdoor_day1_flow.npy')
+    flow_gt = np.load(gt_file)
+    outDict['data']['opticFlow_GT'] = {}
+    outDict['data']['opticFlow_GT']['flowMap'] = {}
+    outDict['data']['opticFlow_GT']['flowMap']['flowMaps'] = np.swapaxes(flow_gt, 1, 3)
+    outDict['data']['opticFlow_GT']['flowMap']['ts'] = ts
+
+    mask_file = os.path.join(os.path.dirname(filePathOrName), 'outdoor_day1_mask.npy')
+    flow_mask = np.load(mask_file)
+    outDict['data']['opticFlow']['mask'] = {}
```

Comment:
Couldn't you treat mask as frame class? Is there a difference between the visualiser of the mask and the frame class? Is it because the mask can be applied over the frame and not the other way around?